src/openloco/map/tile.h

```diff
@@ -109,6 +109,7 @@ namespace openloco::map
         uint8_t base_z() const { return _base_z; }
         uint8_t clear_z() const { return _clear_z; }
 
+        bool is_not_surface() const { return _type & 0x3C; }
```

Comment:
Id prefer if you used `type() != element_type::surface` and not make this a function.